### DISCUSSION TITLE
Approval Workflow allow approvals/denials from email, teams, etc

### DISCUSSION THREAD

**Discussion by Chris530906** [QualiTorque/qualitorque.github.io#259] [Ideas]
As the title says, we have a few customers that would like the ability to approve environments directly from an email or chat channel without needing to go to the Torque UI. I think some kind of hyperlink embedded in the email/chat would suffice.

**Comment by shirabendor-quali** [comment]
Hi @Chris530906 ,
Yes this is something we plan to deliver as part of the roadmap.

Shira
